### ISSUE TITLE
cannot update some documents on beecon.orderofthebee.org

### ISSUE THREAD

**Issue by aviriel** [marsbard/puppet-alfresco#113]
Issue can be reproduced on SOME documents, not on all of them.

Ways to reproduce:
1. Click "Upload new version" on a document in the doclib or on the document page (https://beecon.orderofthebee.org/share/page/site/speakers/document-details?nodeRef=workspace://SpacesStore/f9a654f4-718a-4370-94da-8caed7c9281f in my case, the same I had with presentation templates).
2. Select a file
3. What for the uploading to 100%, then see "Error"
![selection_212](https://cloud.githubusercontent.com/assets/596948/14768212/88afe3f8-0a44-11e6-8174-668955e28aa3.png)
![selection_213](https://cloud.githubusercontent.com/assets/596948/14768214/8b6b2f76-0a44-11e6-81bd-83341b998d70.png)

Trace in the log:
http://pastebin.com/in9GgbeX

Solution: delete file and create the new one with the same name (but it drops the history :( )

**Comment by marsbard** [comment]
Line 179 of the pastebin shows "### Error updating database.  Cause: java.sql.SQLException: Incorrect string value: '\xD0\xA0\xD0\xB0\xD0\xB1...' for column 'qname_localname' at row 1"

it looks like a problem with internationalisation, it doesn't look like it is a specific problem relating to puppet-alfresco.

**Comment by digcat** [comment]
other than should we handle internationalisation?

**Comment by marsbard** [comment]
somehow that error looks like it is confusing Alfresco into thinking it is an upload of a new document hence triggering an error that a file with the same name exists already

**Comment by aviriel** [comment]
yep, it works if I change the language of the browser to english. But I have never seen this errors on other Alfresco installations (4.2 or 5.1), so I'm not sure that it's a problem of generic Alfresco

**Comment by douglascrp** [comment]
I have saw this kind of problem before.
It was fixed when I edited the document title and removed the special
characters.

Em dom, 24 de abr de 2016 12:01, Oksana Kurysheva <notifications@github.com>
escreveu:

> yep, it works if I change the language of the browser to english. But I
> have never seen this errors on other Alfresco installations (4.2 or 5.1),
> so I'm not sure that it's a problem of generic Alfresco
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/marsbard/puppet-alfresco/issues/113#issuecomment-213978841>
>
-- 

Douglas C. R. Paes

**Comment by marsbard** [comment]
In this case it seems to be the other way round, since the filename contains no non-English characters, rather the UI is in Russian. But certainly my first thought was of that occasion that @douglascrp mentions. I tried to replicate the issue on a 4.2.f install by switching my browser language to Russian, but I wasn't able to replicate the issue that @aviriel refers to yet.
